packages/modules/create-imodel/src/components/context/imodel-context.tsx

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React, { createContext, useState } from "react";
+
+import { iModelExtent } from "../../types";
+
+export type iModelProps = {
+  name: string;
+  description: string;
+  thumbnail?: { src?: ArrayBuffer; type: string };
+  extent?: iModelExtent | null;
+};
+
+type ContextProps = {
+  imodel: iModelProps;
+  onPropChange: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => void;
+  onImageChange: (src: ArrayBuffer, type: string) => void;
+  nameString?: string;
```

Comment:
can these be actually undefined? I think they always will have string value.. so you could get rid of all the question marks in your code

Reply:
fixed